CharacterCreation/01_Races.txt

```diff
@@ -72,13 +87,21 @@ requires dim light to see well.
 Sprinter - +1m movement speed when charging, retreating,  or otherwise moving 
 under pressure.
 
-//============================================================
+
+
+==============================
 Detachment A Security Drone
-Society: Slave soldiers. Squad families.
-Security Drones are specialized drones that have been granted a greater AI 
+==============================
+
+===== Society ===== 
+    Slave soldiers. Squad families.

```

Comment:
Missing a space between "Society" and the following line; for consistency.

Reply:
added